README.md

```diff
@@ -12,6 +12,46 @@ You can discuss any issues you have or feature requests in [Discord](https://dis
 
 If you get value out this project and want to show your support you can find me on [patreon](https://www.patreon.com/ddelnano).
 
+## Building provider locally
+
+Requirements:
+* [Go](https://go.dev/doc/install) >= 1.16
```

Comment:
Shouldn't this be >= 1.17? That's what our CI is using.

Reply:
then it's worth to bump versions in [provider](https://github.com/ddelnano/terraform-provider-mikrotik/blob/4b00ace68789b43dbc7d8cbf9d21ce18f7da56e3/go.mod#L3) I suppose?

Reply:
Yea, we should update that. I wish it would be easy to centrally manage the version across all the places it exists.

Since you added a `go.work` file in #115, should we try to bump it to 1.18 if the test suite passes on the upgrade?

Reply:
Will track Go version update by #121